chore/new README.md

edenbd1 opened the pull request.

edenbd1 committed:
new README.md

Le-Caignec reviewed (commented):
What about devTools ?
Do you want to add a schema ?

edenbd1 committed:
Add Dev Tools

Updated README.md to enhance the description of iExec's privacy tools and developer toolkit, and improved community engagement sections.

edenbd1 committed:
Update Use Cases

edenbd1 commented:
I added the Dev Tools and for the schema, i didn’t find one that fits the org yet.
I will ask for one.

Le-Caignec commented on profile/README.md:
```diff
@@ -0,0 +1,70 @@
+| Name | Description | Repo |
+|------|--------------|------|
+| [iApp Generator](https://docs.iex.ec/references/iapp-generator) | CLI tool to scaffold and deploy iExec Confidential Apps | [iExecBlockchainComputing/iapp](https://github.com/iExecBlockchainComputing/iapp) |
+| [Data Protector](https://docs.iex.ec/get-started/helloWorld/2-protectData) | Protect and share data using a decentralized privacy layer | [iExecBlockchainComputing/dataprotector-sdk](https://github.com/iExecBlockchainComputing/dataprotector-sdk) |
```
```suggestion
| [Data Protector](https://docs.iex.ec/references/dataProtector/getting-started) | Protect and share data using a decentralized privacy layer | [iExecBlockchainComputing/dataprotector-sdk](https://github.com/iExecBlockchainComputing/dataprotector-sdk) |
```

Le-Caignec replied:
Can you add a `latest release` column with chips representing the latest SDK version like this? For instance :

<img width="257" height="165" alt="Image" src="https://github.com/user-attachments/assets/1245d307-cdad-4038-87f1-cddd0bfc9c53" />

Le-Caignec commented on profile/README.md:
```diff
@@ -0,0 +1,70 @@
+| ----------------- | ------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------- |
+| [React Starter](https://github.com/iExecBlockchainComputing/iexec-react-starter)     | Using Privy SDK                          | [iExecBlockchainComputing/iexec-react-starter](https://github.com/iExecBlockchainComputing/iexec-react-starter) |
+| [Next.js Starter](https://github.com/iExecBlockchainComputing/iexec-nextjs-starter)   | Using Reown integration | [iExecBlockchainComputing/iexec-nextjs-starter](https://github.com/iExecBlockchainComputing/iexec-nextjs-starter) |
+| [Vue.js Starter](https://github.com/iExecBlockchainComputing/iexec-vuejs-starter)    | Using Reown integration                                | [iExecBlockchainComputing/iexec-vuejs-starter](https://github.com/iExecBlockchainComputing/iexec-vuejs-starter) |
```
links are replicated here - remove the first one on the name

edenbd1 committed:
Add files via upload

Le-Caignec commented on profile/README.md:
```diff
@@ -0,0 +1,70 @@
+
+> [iExec](https://iex.ec) makes it easy to add privacy to your existing apps, or build new ones with privacy at the core.
+> 
+> iExec privacy tools bring encrypted processing, data governance, and trusted execution to your apps, powered by a decentralized network of TEE-based compute.
```
encrypted processing => not true
You can telle instead
iExec privacy tools bring privacy-preserving computation, data governance, and trusted execution to your apps, powered by a decentralized network of TEE-based compute.

edenbd1 committed:
Update Data protector link

Co-authored-by: Robin Le Caignec <72495599+Le-Caignec@users.noreply.github.com>

edenbd1 committed:
Update README

edenbd1 committed:
Add latest release

Added latest release information to developer tools table.
